config/mocha.json

```diff
@@ -10,6 +10,5 @@
     "src/platform/testing/unit/mocha-setup.js",
     "src/platform/testing/unit/enzyme-setup.js"
   ],
-  "reporter": "progress",
```

Comment:
Was this removed intentionally? This is the reporter used when running `yarn test:unit` without the `coverage` option, so it shouldn't interfere with the changes in this PR

Reply:
I think someone else removed that (I took over this ticket). I added it back. Thanks for catching that.